afs/utils.py

```diff
@@ -26,3 +26,9 @@ def _check_response(response):
             except Exception:
                 body = response.text
             raise InvalidStatusCode(response.status_code, body)
+
+def _urljoin(*args, extra_paths):
```

Comment:
When the function name starts with `_`, it means it is an internal function for a module or class. But, `_urljoin()` might be used at anywhere of SDK. I don't think it still an internal function. More detail information at [here](https://stackoverflow.com/a/8689983).

Reply:
Got it. Thanks.